cloud/vsphere/templates.go

```diff
@@ -323,7 +322,6 @@ networking:
   serviceSubnet: ${SERVICE_CIDR}
   podSubnet: ${POD_CIDR}
 kubernetesVersion: v${CONTROL_PLANE_VERSION}
-token: ${TOKEN}
```

Comment:
You should be using the token here no?

Reply:
This is the MasterStartupScript which no longer uses the token.

Reply:
Oh gotcha, this is to register a token with the cluster itself.